src/NewsService.Business/GetNewsCommand.cs

```diff
@@ -15,29 +28,128 @@ public class GetNewsCommand : IGetNewsCommand
     private readonly INewsRepository _repository;
     private readonly INewsResponseMapper _mapper;
     private readonly IHttpContextAccessor _httpContextAccessor;
+    private readonly IRequestClient<IGetUsersDataRequest> _rcGetUsers;
+    private readonly IRequestClient<IGetDepartmentsRequest> _rcGetDepartments;
+    private readonly IDepartmentInfoMapper _departmentInfoMapper;
+    private readonly ILogger<GetNewsCommand> _logger;
+
+    private UserData GetAuthor(Guid userId, List<string> errors)
```

Comment:
make async

Reply:
As far as I know now, it is not necessary to do this yet. We are waiting for a library for this.